aws/resource_aws_wafv2_web_acl.go

```diff
@@ -59,8 +59,8 @@ func resourceAwsWafv2WebACL() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"allow": wafv2EmptySchema(),
-						"block": wafv2EmptySchema(),
+						"allow": wafv2AllowConfigSchema(),
+						"block": wafv2BlockConfigSchema(),
```

Comment:
let's also add these newly defined blocks in wafv2_web_acl.html.markdown

Reply:
These have been added to the markdown now.